component-samples/map-components/tutorials/create-a-web-app-using-components-solution/style.css

```diff
@@ -1,6 +1,6 @@
 /* import the `@arcgis/core` light theme and `@esri/calcite-components` CSS stylesheets. */
-@import "https://js.arcgis.com/4.30/@arcgis/core/assets/esri/themes/light/main.css";
-@import "https://js.arcgis.com/calcite-components/2.8.6/calcite.css";
+@import "https://js.arcgis.com/4.31/@arcgis/core/assets/esri/themes/light/main.css";
+@import "https://js.arcgis.com/calcite-components/2.13.2/calcite.css";
```

Comment:
ditto

Reply:
Can this calcite import be removed too?

Reply:
I'm honestly so confused about the CSS situation right now I don't know.  I don't think the import can be removed if we want the version to sync up with the other SDK tutorials and samples. I'm under the impression we are going to continue to use "newer" versions of calcite in the samples than we do in the core API. I also thought we always had to include the calcite css import when using map components or the pure components like zoom don't work.

Reply:
We can sync up the samples with the doc/tutorials post release.